### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by embray** [astropy/pyregion#145]
Users of newer Python versions are having trouble installing pyregion,  In particular, the pre-cythonized sources in the source tarball are not compatible with Python 3.9 it seems.

This package should probably be updated to a more recent version of the package template and drop astropy_helpers, and maybe include some wheels on PyPI.

I can help take over that work if need be.

**Comment by leejjoon** [comment]
Thanks. I will push a new release soon (I have been away from the practice of maintaining packages for a while so it may take some time to refresh my memories and catch up with new things).

For dropping astropy_helpers: I have started [ape17](https://github.com/astropy/pyregion/tree/ape17) branch which I intend to merge once the transition is done. However, I don't know much about astropy-affiliated packages in fact, and your help will be greatly appreciated. As a baby step, I have pushed a very minor change into that branch.
